### DISCUSSION TITLE
Tab thumbnails

### DISCUSSION THREAD

**Discussion by jerone** [piroor/treestyletab#3399] [Ideas]
The old [Tree Style Tab](https://piro.sakura.ne.jp/xul/_treestyletab.html.en) had support for thumbnails via [Informational Tab](https://piro.sakura.ne.jp/xul/_informationaltab.html.en).

Old screenshots:
![treestyletab_multipletab](https://github.com/piroor/treestyletab/assets/55841/7bafbdc3-3158-43ca-b19c-989a595f7288)
![treestyletab_informationaltab](https://github.com/piroor/treestyletab/assets/55841/eec74959-a162-4603-ba1d-d25f50ea7e26)

Would this be a nice feature for this addon...

Firefox supports getting a screenshot of a tab, via [`tabs.captureTab()`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/captureTab).

**Comment by silverwings15** [comment]
this would be a fantastic addition. here is an implementation for the regular tab bar using an userscript with fx-autoconfig

https://github.com/aminomancer/uc.css.js/blob/master/JS/tabThumbnailTooltip.uc.js
